src/components/App.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+
+import Title from './Title';
+import CustomerReviews from './customerReviews';
+import PriceBox from './PriceBox.js';
+import Stock from './Stock';
+import ItemDescription from './ItemDescription';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    //sample Data
+    this.state = {
+      data: [{
+        "Brand Name": "Oxygen",
+        "Description 1": "Donec dapibus. Duis at velit eu est congue elementum. In hac habitasse platea dictumst. Morbi vestibulum, velit id pretium iaculis, diam erat fermentum justo, nec condimentum neque sapien placerat ante.",
+        "Description 2": "Maecenas tincidunt lacus at velit. Vivamus vel nulla eget eros elementum pellentesque. Quisque porta volutpat erat. Quisque erat eros, viverra eget, congue eget, semper rutrum, nulla.",
+        "Description 3": "Nam dui. Proin leo odio, porttitor id, consequat in, consequat ut, nulla.",
+        "Description 4": "In hac habitasse platea dictumst. Etiam faucibus cursus urna.",
+        "Description 5": "Aliquam quis turpis eget elit sodales scelerisque. Mauris sit amet eros.",
+        "Inventory Amount": 0, 
+        "Is Customer Prime Member?": false,
+        // currently Prime Member does nothing. All shows as Prime
+        "Price": 355,
+        "ProductKey": 1,
+        "Review Total": 1,
+        "Shipping Price": 0,
+        "Title": "Pannewitz's Sedge",
+        "Total 1*": 1,
+        "Total 2*": null,
+        "Total 3*": null,
+        "Total 4*": null,
+        "Total 5*": null,
+        "Total Review Stars": 1
```

Comment:
Is it necessary to have hard coded data that loads initially? It makes the code harder to read.

Reply:
Cool, it's just for testing! I can remove on final.